src/main/java/eu/darkbot/utils/ThrowableItem.java

```diff
@@ -0,0 +1,5 @@
+package eu.darkbot.utils;
+
+public interface ThrowableItem {
```

Comment:
*Throwable* is a terrible word to use for anything that is not a type of exception or related. Maybe CastableItem? UsableItem? ConsumableItem ? or something like that